core/src/com/unciv/ui/victoryscreen/RankingManager.kt

```diff
@@ -0,0 +1,22 @@
+package com.unciv.ui.victoryscreen
+
+import com.unciv.logic.civilization.CivilizationInfo
+import kotlin.math.roundToInt
+
+class RankingManager(private val civInfo: CivilizationInfo) {
```

Comment:
I don't think this needs to be a class? why not make the method in this class simply an extension method on civInfo or instead of making units not private, have the logic inside the instance method instead?

Also, I suggest a different name besides getRanking since that's not actually what it's getting.  It's getting aCiv Summary Stat or something.

Reply:
I also thought about the extension method, however, this class can grow in a future with other functions similar functions. Maybe, you are right and we can limit ourselves with a just function for now.

Reply:
With regard to naming, I don't like naming it as Civ Stat in order to not mess up with existing `Stats` which are used to represent current production, science, etc.
This representation includes not only `Stats` but also other aspects from other entities like Power of Army or population of the cities.